fem/fespace.hpp

```diff
@@ -106,6 +108,11 @@ class FiniteElementSpace
 
    int nvdofs, nedofs, nfdofs, nbdofs;
    int *fdofs, *bdofs;
+   /** These variables store the number of Interior and Boundary faces.
+     * fes->GetMesh()->GetNBE() doesn't return the expected value in 3D
+     * because periodic meshes in 3D have some faces marked as boundary
+     * for vizualization purpose in GLvis. */
+   mutable int nbInteriorFaces, nbBoundaryFaces;
```

Comment:
These are properties of the `Mesh` -- it makes sense to move them there.

Reply:
done.